metrics-monitoring/README.adoc

```diff
@@ -0,0 +1,225 @@
+* You or other users have produced and consumed messages in the Kafka topics. For example, to use Kafka scripts to produce and consume messages, see {base-url}{kafka-bin-scripts-url}[_Configuring and connecting Kafka scripts with {product-long}_^].
+
+.Procedure
+* In the *Kafka Instances* page of the web console, click the name of the Kafka instance and select the *Dashboard* tab.
++
+--
+When you create your Kafka instance and topics, the *Dashboard* page is initially empty. After you start producing and consuming messages in your services, you can return to this page to view related metrics.
+
+NOTE: In some cases, after you start producing and consuming messages, you might need to wait several minutes for the latest metrics to appear. You might also need to wait until your instance and topics contain enough data for metrics to appear.
+
+--
+
+[id="proc-configuring-metrics-prometheus_{context}"]
+== Configuring metrics monitoring for a Kafka instance in Prometheus
+
+As an alternative to viewing metrics for a Kafka instance in the {product} web console, you can export your metrics to https://prometheus.io/docs/introduction/overview/[Prometheus] and integrate the metrics with your own metrics monitoring platform. {product} provides a `kafkas/{id}/metrics/federate` API endpoint that you can configure as a scrape target for Prometheus to use to collect and store metrics. You can then access the metrics in the https://prometheus.io/docs/visualization/browser/[Prometheus expression browser] or in a data-graphing tool such as https://prometheus.io/docs/visualization/grafana/[Grafana].
+
+This procedure follows the https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/additional-scrape-config.md#additional-scrape-configuration[Additional Scrape Configuration] method defined by Prometheus for integrating third-party metrics.
+
+.Prerequisites
+* You have access to a Kafka instance that contains topics in {product}. For more information about access management in {product}, see {base-url}{access-mgmt-url}[_Managing account access in {product-long}_^].
+* You have the generated credentials for your service account that has access to the Kafka instance. To regenerate and re-copy the credentials, use the *Service Accounts* page in the {product} web console to find your service account and update the credentials. For more information about service accounts, see {base-url}{getting-started-url}#proc-creating-service-account_getting-started[_Getting started with {product-long}_^].
+* You or other users have produced and consumed messages in the Kafka topics. For example, to use Kafka scripts to produce and consume messages, see {base-url}{kafka-bin-scripts-url}[_Configuring and connecting Kafka scripts with {product-long}_^].
```

Comment:
I don't really see this as a prerequisite of configuring the scrape endpoint in Prometheus. This can still be set up even if the Kafka instance hasn't been used yet (no connections, no topics, etc.). The metrics around those may not appear until then but the scrape endpoint will still be up and running. Once Kafka is used, those metrics should then appear/result updated.

Reply:
Yes, great point. For both this and the other procedure, I've removed that prereq and moved it down into the actual steps as a mention of expected behavior when users first create new instances and topics. Thanks.